### PR TITLE
Adding migrations bundle

### PR DESCRIPTION
**Pull request by ryandillinfelton**
Adding a new bundle (migration) and a tile on the landing page for it

### CODE REVIEW

File: src/js/App/Header/InsightsAbout.js [hunk 1]
```diff
@@ -73,7 +73,9 @@ class InsightsAbout extends Component {
                 { name: 'Compliance', path: 'apps/compliance/app.info.json', version: 'N/A' },
                 { name: 'Cost Management', path: 'apps/cost-management/app.info.json', version: 'N/A' },
                 { name: 'Insights', path: 'apps/insights/app.info.json', version: 'N/A' },
-                { name: 'System Comparison', path: 'apps/drift/app.info.json', version: 'N/A' }
+                { name: 'System Comparison', path: 'apps/drift/app.info.json', version: 'N/A' },
+                //concerned about this bit
+                { name: 'Migration', path: 'migrations/migration-analytics/app.info.json', version: 'N/A' }
```
Comment (ryelo) [inline src/js/App/Header/InsightsAbout.js]: Let's call this "Migrations" just for consistency

File: src/js/jwt/insights/user.js [hunk 2]
```diff
@@ -5,7 +5,8 @@ const pathMapper = {
     rhel: 'smart_management',
     hybrid: 'hybrid_cloud',
     insights: 'insights',
-    openshift: 'openshift'
+    openshift: 'openshift',
+    migration: 'migration'
```
Comment (ryelo) [inline src/js/jwt/insights/user.js]: Same thing here

File: src/js/App/Header/InsightsAbout.js (hunk 1 above)
Comment (ryandillinfelton) [reply src/js/App/Header/InsightsAbout.js]: fixed

File: src/js/jwt/insights/user.js (hunk 2 above)
Comment (ryandillinfelton) [reply src/js/jwt/insights/user.js]: fixed

File: src/js/App/Header/InsightsAbout.js [hunk 3]
```diff
@@ -73,7 +73,9 @@ class InsightsAbout extends Component {
                 { name: 'Compliance', path: 'apps/compliance/app.info.json', version: 'N/A' },
                 { name: 'Cost Management', path: 'apps/cost-management/app.info.json', version: 'N/A' },
                 { name: 'Insights', path: 'apps/insights/app.info.json', version: 'N/A' },
-                { name: 'System Comparison', path: 'apps/drift/app.info.json', version: 'N/A' }
+                { name: 'System Comparison', path: 'apps/drift/app.info.json', version: 'N/A' },
+                //concerned about this bit
+                { name: 'Migrations', path: 'migrations/migration-analytics/app.info.json', version: 'N/A' }
```
Comment (ryelo) [inline src/js/App/Header/InsightsAbout.js]: I think this path is actually going to be `apps/migration-analytics/app.info.json`

They're in beta right now, so you can see the path:
https://ci.cloud.paas.psi.redhat.com/beta/apps/migration-analytics/app.info.json

Comment (ryandillinfelton) [reply src/js/App/Header/InsightsAbout.js]: Thanks! I was concerned about that bit

**Comment by kruai** [comment]
In the Source of Truth, the ID for the Migration Analytics app is `migration-analytics`; in this PR, it's `migrations`. Do you know if they prefer one way or the other?

File: src/js/App/Header/InsightsAbout.js [hunk 4]
```diff
@@ -73,7 +73,8 @@ class InsightsAbout extends Component {
                 { name: 'Compliance', path: 'apps/compliance/app.info.json', version: 'N/A' },
                 { name: 'Cost Management', path: 'apps/cost-management/app.info.json', version: 'N/A' },
                 { name: 'Insights', path: 'apps/insights/app.info.json', version: 'N/A' },
-                { name: 'System Comparison', path: 'apps/drift/app.info.json', version: 'N/A' }
+                { name: 'System Comparison', path: 'apps/drift/app.info.json', version: 'N/A' },
+                { name: 'Migrations', path: 'apps/migration-analytics/app.info.json', version: 'N/A' }
```
Comment (ryandillinfelton) [inline src/js/App/Header/InsightsAbout.js]: @kruai should this line instead be
`{ name: 'Migration Analytics', path: 'apps/migration-analytics/app.info.json', version: 'N/A' }` ?

**Comment by kruai** [comment]
> > In the Source of Truth, the ID for the Migration Analytics app is `migration-analytics`; in this PR, it's `migrations`. Do you know if they prefer one way or the other?
> 
> 'migration-analytics' should still be the name of the one app in the new 'migrations' bundle. If there is a spot where I should be referring to the app name instead of the bundle name then that's a mistake, not an attempt at renaming the app itself.

Ah, I see what you mean now that I'm taking a look at the cloud-services-config PR.

File: src/js/App/Header/InsightsAbout.js (hunk 4 above)
Comment (kruai) [reply src/js/App/Header/InsightsAbout.js]: Yes, it probably should!

**Pull request merged**